easybuild/easyblocks/d/doxygen.py

```diff
@@ -30,30 +30,36 @@
 @author: Kenneth Hoste (Ghent University)
 @author: Pieter De Baets (Ghent University)
 @author: Jens Timmerman (Ghent University)
+@author: Balazs Hajgato (Free University Brussels (VUB))
 """
 
-from easybuild.easyblocks.generic.configuremake import ConfigureMake
+from distutils.version import LooseVersion
 from easybuild.tools.run import run_cmd
+from easybuild.easyblocks.generic.cmakemake import CMakeMake
 
-
-class EB_Doxygen(ConfigureMake):
+class EB_Doxygen(CMakeMake):
     """Support for building/installing Doxygen"""
 
+    version_switch = LooseVersion(self.version) < LooseVersion("1.8.10")
```

Comment:
I mean where and how? Apparently 3rd works with the old (but then the EasyBlock is identical to the old), but not for the new.